contentcuration/contentcuration/settings.py

```diff
@@ -23,6 +23,8 @@
 from contentcuration.utils.incidents import INCIDENTS
 from contentcuration.utils.secretmanagement import get_secret
 
+ADMIN_ENABLED = True
```

Comment:
Because the consequences of this setting are used within the same settings file, overriding it in the derived settings is not having the effect you intend.

I would recommend just removing this setting completely, and doing the simpler change of just moving the django admin urls from `urls.py` to `dev_urls.py`

Reply:
Done